platform/darwin/include/mbgl/interface/native_apple_interface.h

```diff
@@ -0,0 +1,46 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLNativeAppleInterfaceDelegate <NSObject>
+
+@optional
+
+- (NSString *)nai_skuToken;
+- (NSString *)nai_accountTypeKey;
```

Comment:
We no longer need account type here, so let's remove.

Reply:
Could u verify this? @julianrex @tmpsantos 
I saw we are still using the `accountTypeKey` in `http_file_source.mm`, for example:
```
  if (accountType == 0 && isValidMapboxEndpoint(url)) {
        NSURLComponents *components = [NSURLComponents componentsWithURL:url resolvingAgainstBaseURL:NO];
        NSMutableArray *queryItems = [NSMutableArray array];
        
        if (resource.usage == Resource::Usage::Offline) {
            [queryItems addObject:[NSURLQueryItem queryItemWithName:@"offline" value:@"true"]];
        } else {
            // Only add SKU token to requests not tagged as "offline" usage.
            [queryItems addObject:[NSURLQueryItem queryItemWithName:@"sku" value:MGLNativeAppleInterfaceTransmitter.shared.skuToken]];
        }
        
        if (components.queryItems) {
            [queryItems addObjectsFromArray:components.queryItems];
        }
        
        components.queryItems = queryItems;
        url = components.URL;
    }
```

If so, we should modify all calls of `accountKeyType` in this file.

Reply:
`accountTypeKey` is a constant. We use to build a valid URL with parameters. Has this changed @julianrex ?

Reply:
Yes, we can remove, and can remove the `accountType == 0` check from the `if` too.